docs/getting-started/index.md

```diff
@@ -8,58 +8,58 @@ meta-description: Learn how to install, integrate, configure, and develop CKEdit
+
+CKEditor&nbsp;5 is also compatible with popular CSS frameworks such as Bootstrap or Foundation. Such integrations, however, often require additional changes and adjustments that we have gathered {@link getting-started/integrations/css in this guide}.
 
 We plan to provide more integrations with time. We would like to [hear your ideas](https://github.com/ckeditor/ckeditor5/issues/1002) about what we should work on next.
 
-### How do I use CKEditor&nbsp;5 if my framework does not have an official integration?
+<!-- ### How do I use CKEditor&nbsp;5 if my framework does not have an official integration?
 
 CKEditor&nbsp;5 should be compatible with your framework and initializing it usually requires a single method call. A tighter integration of CKEditor&nbsp;5 with your framework may require using an existing or writing a new adapter (integration layer) to communicate your framework with CKEditor&nbsp;5.
 
 When checking how to integrate CKEditor&nbsp;5 with your framework, you can follow these steps:
 
-1. **Check whether an [official integration](#official-wysiwyg-editor-integrations) exists.**
+* If no official integrations (listed above) are available, search for community-driven integrations. Most of them are available on [npm](https://www.npmjs.com/).
+* If none exists, integrate CKEditor&nbsp;5 with your framework by yourself.
 
-	There are four official integrations now: for {@link getting-started/integrations/react React}, {@link getting-started/integrations/angular Angular}, {@link getting-started/integrations/vuejs-v2 Vue.js 2.x}, and {@link getting-started/integrations/vuejs-v3 Vue.js 3.x}.
-2. **If not, search for community-driven integrations.** Most of them are available on [npm](https://www.npmjs.com/).
-3. **If none exists, integrate CKEditor&nbsp;5 with your framework by yourself.**
+CKEditor&nbsp;5 offers {@link getting-started/legacy-getting-started/predefined-builds predefined builds} that expose a rich JavaScript API, which you can use to {@link getting-started/legacy-getting-started/editor-lifecycle#creating-an-editor-with-create create editors} and {@link getting-started/getting-and-setting-data#setting-the-editor-data-with-setdata control them}. -->
+
+## Configuring CKEditor&nbsp;5
+
+Once you install your copy of CKEditor&nbsp;5, take some time to {@link getting-started/setup/configuration configure it} before first use. Set up data saving methods, editor toolbars and UI.
+
+You may also take the {@link tutorials/crash-course/editor step-by-step tutorial} that will guide you through the installation and configuration of the editor.
+
+## Want to remove the "Powered by CKEditor" logo?
```

Comment:
I get that people need such a how-to. But I wonder if it's so important that we need to put it in the overview.

Reply:
It's like first point of contact, that's why we had it on top for the past monts (and still have). I judged we can keep it, just not that exposed as before.